src/lib/operators/import.cpp

```diff
@@ -51,17 +53,49 @@ std::shared_ptr<const Table> Import::_on_execute() {
   auto table = std::shared_ptr<Table>{};
 
   switch (_file_type) {
-    case FileType::Csv:
-      table = CsvParser::parse(filename, _chunk_size, _csv_meta);
+    case FileType::Csv: {
+      if (static_cast<int>(static_cast<bool>(_csv_meta)) +
+              static_cast<int>(Hyrise::get().storage_manager.has_table(_tablename)) +
+              static_cast<int>(std::filesystem::exists(filename + CsvMeta::META_FILE_EXTENSION)) >=
```

Comment:
Can we have much simple checks here? Performance does not matter.

If _csv_meta, check for table existing or fail.

If no _csv_meta, check that meta file exists and table does not. Be more explicit.

Reply:
I feel like this differs from what we discussed in person. Maybe it makes sense to talk about this in a call instead of here?